rudder-lang/tools/generate_lib

```diff
@@ -3,6 +3,41 @@
 use warnings;
 use strict;
 
+sub get_paths_from_conf_file {
+  my ($file) = @_;
+
+  my $is_default = 0;
+  my $ncf = "/usr/share/ncf/tree/30_generic_methods/";
+  my $dsc_ncf = "/var/rudder/configuration-repository/dsc/ncf/30_generic_methods/";
+  my $resourcelib_path = "/opt/rudder/share/rudder-lang/lib";
+  my $generic_methods = "/opt/rudder/share/rudder-lang/tools/generic_methods.toml";
+  open (FILE, $file);
+  while (<FILE>) {
+    chomp;
+    if ($is_default == 1 && /^\s*ncf\s*=\s*"(?<path>[\/\w\.\-\~]+)"/ ) {
```

Comment:
/,~ and . don't need to be backslashed in regex

Reply:
if you accept ~ explicitly, does it mean you support home directory expansion ?

Reply:
Ok using tilde would force to import perl libraries + their dependencies. Therefore I remove it, forcing users to write full path